python/invoke_release/tasks.py

```diff
@@ -1503,3 +1512,22 @@ def wheel(_):
         archive_name=archive_name,
         base_dir=base_dir
     ))
+
+
+def create_pull_request(options, user, token, title,
+                        github_url='https://api.github.com'):
+    headers = {"Authorization": "token %s" % token,
```

Comment:
Please use single quotes and `'token {}'.format(token)` instead of the `%` syntax.